packages/stylelint-config-react-app/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = {
+  plugins: ['stylelint-csstree-validator'],
+  rules: {
+    'at-rule-no-unknown': true,
+    'color-no-invalid-hex': true,
+    'declaration-block-no-duplicate-properties': true,
```

Comment:
Wouldn't be better to use [declaration-block-no-duplicate-properties](https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties/) with `ignore: ["consecutive-duplicates"]`?

Moreover, is this rule blocks the pattern of (Added example)?
```
margin: 1rem;
margin-bottom: 2rem;
```